tools/realsense-viewer/CMakeLists.txt

```diff
@@ -196,6 +211,10 @@ if(OPENVINO2019)
     target_compile_definitions(realsense-viewer PRIVATE OPENVINO2019)
 endif()
 
+if(OPENVINO_NGRAPH)
```

Comment:
Shouldn't this be in an elseif(), like elsewhere?

Reply:
no. this is to define OPENVINO_NGRAPH compilation definition if ngraph presents in the openvino version.